specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/dscpConfiguration.json

```diff
@@ -0,0 +1,343 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2020-06-01"
```

Comment:
>2020-06-01 [](start = 16, length = 10)

Has the `2020-06-01` been released to customers?

Reply:
By the end of this month it should be supported, as far as I'm aware. Did notice yesterday while doing a little bit of support that NRP is already supporting in production this version.